include/benchmark/reporter.h

```diff
@@ -105,12 +110,45 @@ class BenchmarkReporter {
   // reported.
   virtual void Finalize();
 
+  // REQUIRES: 'out' shall be valid for the lifetime of the reporter.
+  // Set the stream used for non-error output to the value specified by 'out'.
+  void SetOutputStream(std::ostream& out) {
```

Comment:
google style: no non-const references. please take a pointer directly (and CHECK for nullptr. sorry).